src/client/Database/DatabaseDefinition.ts

```diff
@@ -1,4 +1,6 @@
 export interface DatabaseDefinition {
   /** The id of the database. */
   id?: string;
+  /** Throughput for this database. */
+  throughput?: number;
```

Comment:
What if we split the types instead? Having the same type for the request payload and the response body seems ripe for future issues like this.

Reply:
You can add another type and intersect it on the request. That's another option. Not sure what I'd name it.

Reply:
Or just extend definition.

Reply:
I made `DatabaseCreateRequest` and `ContainerCreateRequest` since these can't yet be set on update

Reply:
Now I'm not sure if that is the best route. It would be nice if we could use `DatabaseRequest` without a name change for create/update even if we can't right now.